java/java.d.ts

```diff
@@ -79,1337 +42,10 @@ declare module Java {
     ensureJvm(done: Callback<void>): void;
     ensureJvm(): Promise<void>;
 
-    newShort(val: number): java.lang.Short;
```

Comment:
I guess these number factory methods are an acceptable casualty of this refactor.